src/zm_frame.h

```diff
@@ -20,12 +20,16 @@
 #ifndef ZM_FRAME_H
 #define ZM_FRAME_H
 
+#include "zm_time.h"
 #include <sys/time.h>
-#include <sys/types.h>
-class Frame;
 
-#include "zm_event.h"
-#include "zm_time.h"
+typedef uint64_t event_id_t;
```

Comment:
why move this to here?  It is defining the event id type.  I think it should stay in zm_event.h

Reply:
Good catch. I moved the typedef now back and solved the circular dependency which triggered this changes differently.